pkg/api/handlers/compat/exec.go

```diff
@@ -104,15 +123,6 @@ func ExecInspectHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	utils.WriteResponse(w, http.StatusOK, inspectOut)
```

Comment:
```suggestion
	utils.WriteResponse(w, http.StatusCreated, inspectOut)
```
From https://docs.docker.com/engine/api/v1.40/#operation/ContainerExec

Reply:
This isn't Exec, this is ExecStart, which is a 200